frontend/src/metabase/dashboard/components/DashboardInfoSidebar/components/DashboardEntityIdCard.unit.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import { getMantineSelectOptions } from "__support__/components/mantineSelect";
+import { renderWithProviders, screen } from "__support__/ui";
+import {
+  createMockDashboard,
+  createMockDashboardTab,
+} from "metabase-types/api/mocks";
+
+import { DashboardEntityIdCard } from "./DashboardEntityIdCard"; // Adjust the import path as needed
+
+describe("DashboardEntityIdCard", () => {
+  it("should display all tabs from the dashboard as options in the Select", async () => {
+    const dashboard = createMockDashboard({
+      tabs: [
+        createMockDashboardTab({ id: 1, name: "Tab 1" }),
+        createMockDashboardTab({ id: 2, name: "Tab 2" }),
+        createMockDashboardTab({ id: 3, name: "Tab 3" }),
+      ],
+    });
+    renderWithProviders(<DashboardEntityIdCard dashboard={dashboard} />);
+
+    const { optionTextContents, displayedOption } =
+      await getMantineSelectOptions();
+    expect(optionTextContents).toEqual(["Tab 1", "Tab 2", "Tab 3"]);
+    expect(displayedOption.value).toBe("Tab 1");
```

Comment:
can we test that the entity ids show up? and that the select changes which entity id appears?